docs/bare_metal/using_bare_metal.md

```diff
@@ -9,6 +9,8 @@ Here is a code snippet that can work for both Mbed OS profiles; it prints text a
 
 [![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_RTOS/EventQueue_ex_2/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_RTOS/EventQueue_ex_2/main.cpp)
 
+<span class="notes">**Note:** To be compatible with Arm microlib, a bare metal application should not return from `main()`. In this example the `queue.dispatch_forever()` call never returns. For more details, see [Non-returning main() required](../baremetal/using-small-c-libraries.html#non-returning-main-required).</span>
```

Comment:
```suggestion
<span class="notes">**Note:** To be compatible with Arm microlib, a bare metal application should not return from `main()`. In this example, the `queue.dispatch_forever()` call never returns. For more details, see [Non-returning main()](../baremetal/using-small-c-libraries.html#non-returning-main).</span>
```

Reply:
For the record, I don't know if the link will work when the header has `()`. Please open all future PRs from a branch, not a fork, so I can have a preview.

Reply:
My VS Code previewer works with `()`, hopefully it's standard. I'll use branches for future PRs.